lib/reporter/logger.js

```diff
@@ -11,14 +10,13 @@ function logModule(log, logType, module) {
   }
   if (module.error) {
     log[logType]('error:', module.error.message);
-    log[logType]('error:', module.testOutput);
+    // If the output is too big the log processor explodes.
+    log[logType]('error:', module.testOutput.substring(0, 10000));
```

Comment:
could we iterate over substrings rather than drop arbitrary content?

Reply:
Ack.
I had the same thought. Definitely worth a try.

Reply:
Done